### PR TITLE
Update sample workload xnnpack

### PR DESCRIPTION
**Pull request by TianlongLiang**
Add a shell script for building xnnpack examples and run it

### CODE REVIEW

File: samples/workload/XNNPACK/build.sh [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+# 1. build xnnpack wasm files
+# 1.1 to decide whether needs to build xnnpack wasm files
+cd ${WAMR_DIR}/samples/workload/XNNPACK
+xnnpack_out_dir=xnnpack/bazel-bin
+if [ -d "$xnnpack_out_dir" ]; then
+  echo "XNNPACK build directory exists"
+  file_count=$(ls "$xnnpack_out_dir" | grep ".wasm$" | wc -l)
+  # TODO: currently should be 116 files, may changes in the future
+  if [ $file_count -eq 116 ]; then
```
Comment (lum1n0us) [inline samples/workload/XNNPACK/build.sh:28]: `egrep "\/\/:.*\.wasm" CMakeLists.txt | wc` can get the amount of .wasm

File: samples/workload/XNNPACK/build.sh [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  cmake --build build
+fi
+
+# 1.3 copy files to out/
+rm -rf ${OUT_DIR}
+mkdir ${OUT_DIR}
+cp xnnpack/bazel-bin/*.wasm out/
+
+# 2. compile xnnpack benchmark wasm files to benchmark aot files with wamrc
+# 2.1 build wamr-compiler if needed
```
Comment (lum1n0us) [inline samples/workload/XNNPACK/build.sh:56]:
```suggestion
# 2.1 build wamr-compiler and llvm libraries if needed
```

File: samples/workload/XNNPACK/build.sh [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+cd ${WAMRC_DIR}
```
Comment (lum1n0us) [inline samples/workload/XNNPACK/build.sh:57]: maybe some `test` first to achieve "build if needed"

Comment (lum1n0us) [reply samples/workload/XNNPACK/build.sh:57]: And if *.wasm -> *.aot is necessary, we shall add it in *CMakeLists.txt*

Comment (TianlongLiang) [reply samples/workload/XNNPACK/build.sh:57]: SIMD feature is only supported on aot mode, so maybe it's a good idea to add steps `building wamrc` and `translate wasm to aot` to CMakeLists.txt?

**Pull request closed without merging**